### ISSUE TITLE
Playlists are too easy to reorganize 

### ISSUE THREAD

**Issue by realityfabric** [Chocobozzz/PeerTube#2078]
**What happened?**

Trying to "scroll" on (self created) playlist pages is very difficult on a phone, because the videos can be reorganized by dragging from any location on the video tile. Because scrolling on a phone involves dragging, this leaves only the thin margin to the left of the video tiles, and I inevitably end up moving videos around. 

**What do you expect to happen instead?**

Having a "handle" to click and drag, instead of the whole video tile, would be nice. 

**Steps to reproduce:**
1. Add enough videos to a playlist that you'd need to scroll. 6 or 7 should be enough.
2. Use a phone to open your playlist while logged in. 
3. Try to move up and down the page.

**Additional information**
* PeerTube version or URL: v1.4.0-rc.1
* Browser name/version: Firefox for Android

**Comment by alcalyn** [comment]
I submitted a PR to disable drag and drop on small screens. That fixes the UI bug, but we cannot anymore reorder on small screen (enlarging screen enable it again).

Should I add a handle to drag and drop videos, but it adds an icon and use more space as we are on a small screen, or maybe a "Reorder" button to enable drag and drop only when wanted ?

**Comment by alcalyn** [comment]
Finally I tried to add a button to enable/disable reordering, that was not so nice and it does not fixe the problem (still being stuck when at the middle of the playlist).

I tried handlers but couldn't because handle on element would not be on the same angular component as the draggable list, also the handle take to much place on already small screen.

So I tried with a 500ms delay, and seems to give the best UX.
